cinema/serializes.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` instead of `serializers.Serializer`. This will automatically generate the `create` and `update` methods based on the model fields, reducing boilerplate code.